src/edu/wright/cs/raiderplanner/controller/SettingsController.java

```diff
@@ -0,0 +1,412 @@
+	 * May be assigned to a button in the initialize method and
+	 * what it does may be applied in the main method's switch statement.
+	 */
+	public enum Window {
+		EMPTY, ABOUT, GENERAL, ACCOUNT, THEME, NOTIFICATIONS
+	}
+
+	private Window current;
+	private boolean isNavOpen;
+	private boolean mouseDown = false;
+	private boolean initialLoad = true;
+
+	// Screen size:
+	private double screenWidth = Screen.getPrimary().getVisualBounds().getWidth();
+	private double screenHeight = Screen.getPrimary().getVisualBounds().getHeight();
+	private double screenAverage = (screenWidth + screenHeight) / 2;
+
+	// Shadows:
+	private int navShadowRadius = (int) (screenAverage * 0.03);
+	private int navShadowOffset = (int) (screenAverage * 0.01);
+	private DropShadow navShadow = new DropShadow(navShadowRadius, navShadowOffset, 0, Color.BLACK);
+	//private DropShadow notifShadow = new DropShadow(screenAverage * 0.02, 0, 0.009, Color.BLACK);
+	private DropShadow moduleDefaultShadow = new DropShadow(screenAverage * 0.005, 0, 0,
+			Color.BLACK);
+	private DropShadow moduleHoverShadow = new DropShadow(screenAverage * 0.02, 0, 0, Color.BLACK);
+	private InnerShadow modulePressedShadow = new InnerShadow(screenAverage * 0.017, 0, 0,
+			Color.BLACK);
+
+	// Labels:
+	private Label welcome;
+	@FXML
+	private Label title;
+
+	// Buttons:
+	@FXML
+	private Button openMenu;
+	@FXML
+	private Button generalBtn;
+	@FXML
+	private Button accountBtn;
+	@FXML
+	private Button themeBtn;
+	@FXML
+	private Button notificationsBtn;
+	@FXML
+	private Button aboutBtn;
+	@FXML
+	private Button closeDrawer;
+
+	// Panes:
+	@FXML
+	private AnchorPane navList;
+	@FXML
+	private GridPane mainContent;
+	@FXML
+	private HBox topBox;
+	@FXML
+
+	//chat variables
+	private final BorderPane mainPane = new BorderPane();
+
+	/**
+	 * Sets this.current to equal passed variable and calls this.main().
+	 */
+	public void main(Window wind) {
+		this.current = wind;
+		this.main();
+	}
+
+	/**
+	 * Main method containing switch statements.
+	 * Switch statements detect what state the app is in.
+	 */
+	public void main() {
+		if (isNavOpen) {
+			openMenu.fire();
+		}
+
+		initialLoad = false;
+
+		switch (this.current) {
+		case ABOUT: {
+			loadAbout();
+			break;
+		}
+		case GENERAL: {
+			this.loadGeneral();
+			break;
+		}
+		case ACCOUNT: {
+			this.loadAccount();
+			break;
+		}
+		case THEME: {
+			this.loadTheme();
+			break;
+		}
+		case NOTIFICATIONS: {
+			this.loadNotification();
+			break;
+		}
+		default:
+			break;
+		}
+	}
+
+	/**
+	 * Display the About menu.
+	 */
+	public void loadAbout() {
+		// set ToolTips
+		this.mainContent.getChildren().remove(1, this.mainContent.getChildren().size());
+		this.topBox.getChildren().clear();
+		this.title.setText("About");
+
+		//Text Labels:
+		Label appName = new Label("RaiderPlanner");
+		appName.setFont(Font.font("Ariel", FontWeight.BOLD , 42));
+
+		Label versionNo = new Label("Version 1.0.1\nCopyright © 2017");
```

Comment:
I don't think that we have an official version number, so just go with 0.0.0 for now.

Reply:
I can change that.